packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -839,7 +842,8 @@ class Auth {
     async register(body: ShopperCustomersTypes.CustomerRegistration) {
         const {
             customer: {login},
-            password
+            password,
+            ...customParameters
```

Comment:
customParams for the body also needs to follow c_* pattern. Otherwise, it will be rejected. See [here](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-login?meta=authenticateCustomer)